common/analysis/lint_waiver.cc

```diff
@@ -376,6 +377,24 @@ static absl::Status WaiveCommandHandler(
           waiver->WaiveLineRange(rule, line_start - 1, line_end);
         }
 
+	if (!can_use_regex && !can_use_lineno) {
+	  unsigned int number_of_lines = 0;
+	  std::string line;
+	  std::ifstream user_file(lintee_filename.data());
+
+	  if (!user_file.is_open()) {
+	    return WaiveCommandError(token_pos, waive_file,
+				     "lintee_filename was not opened");
+	  }
+
+	  while (std::getline(user_file, line)) {
+	    number_of_lines++;
+	  }
+
+	  user_file.close();
```

Comment:
Recommend using:

https://github.com/google/verible/blob/1886f72085356d3169d35a8b496f97fe1c662959/common/util/file_util.h#L45

to get file contents, and then `std::count(contents.begin(), contents.end(), '\n')` to count lines.